### ISSUE TITLE
When page is reflowed content is not fully visible

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#130]
URL:  https://recalls-rappels.canada.ca/fr/avis-rappel/rappel-certains-produits-vegetaliens-et-vegetariens-marque-happy-veggie-world-hearts
User Agent: Chrome v98
Language: English & French

Issue Description: When the content is reflowed, some content is not fully visible and there is overlap.

Steps to reproduce:
1. Navigate to the url above.  Or Page name for the test is: 'Certain Happy Veggie World, Hearts Choices, and Vegefarm brand vegan and vegetarian products recalled due to undeclared egg' in case it needs to be searched on for next test.
2. Set the desktop browser to 1280(W) X 1024(H) and zoom in to 400% (Chrome -> Web Developer settings ->Resize Window to 1280X1024).

Expected Result: When content is zoomed up to 400% the content should be presented without loss of information or text overlap. 

Please note that data tables are allowed 2 dimensional scrolling.

Note to tester: When re-testing this ticket ensure that the wrapping under Advisory details gets corrected when in normal view.

![image](https://user-images.githubusercontent.com/68117281/159055914-5d32319b-3acc-4490-962c-036816528aca.png)

![image](https://user-images.githubusercontent.com/68117281/159055925-b7ef8152-a981-45c8-b76e-c3dadeae767f.png)
![image](https://user-images.githubusercontent.com/68117281/159055942-111c4a11-87da-4b89-ada5-ca5827e40313.png)
![image](https://user-images.githubusercontent.com/68117281/159055984-04074cf1-83ec-41d3-aa66-95120edaa965.png)

**Comment by ndolter** [comment]
Page name for the test is: 'Certain Happy Veggie World, Hearts Choices, and Vegefarm brand vegan and vegetarian products recalled due to undeclared egg' in case it needs to be searched on for next test.

**Comment by davidgotlieb** [comment]
This is actually a bug. Have reported it so it can be fixed. It relates to old migrated content not displaying properly. It shouldn't be in two columns at the top like that.

**Comment by davidgotlieb** [comment]
Bug has been fixed. Thanks for flagging.

**Comment by ndolter** [comment]
The issue with the wrapping paragraph has been fixed, however, the overlapping text in reflow is still an issue.

**Comment by echouin** [comment]
@ndolter can you please test with a Post-November 2021 recall to verify?

Otherwise, this will be tagged as "Legacy" item as per my convo w/ @mmiron-HC

**Comment by ndolter** [comment]
@echouin I checked the below 2 pages in both languages and there was no issue with reflow:
- https://recalls-rappels.canada.ca/en/alert-recall/wondertoys-27-pcs-wooden-magnetic-alphabet-train-set-recalled-due-presence-superglue
- https://recalls-rappels.canada.ca/en/alert-recall/urban-kids-plush-all-over-print-robes-recalled-due-flammability-hazard

**Comment by echouin** [comment]
@ndolter so is that another one closed?

**Comment by ndolter** [comment]
@echouin Manon requested the following in https://github.com/mmiron-HC/WST-RSAMS/issues/134#issuecomment-1075463418: Can Milestone be "legacy: not fixed"? I typically sort that way when auditing.

**Comment by mmiron-HC** [comment]
@zimw7 @YuxuanXie01 @40999221 Can you please retest. If it fails, please move over to the ART epic

**Comment by YuxuanXie01** [comment]
Retested, issue resolved.

**Comment by 40999221** [comment]
Issue with wrapping paragraph is fixed but have some reflow issue as table used for page layout. 
1) Recalled products - text coming close and 2) Media enquiries  as showing in the screenshot below:

1) text coming close

![image](https://github.com/user-attachments/assets/a4ed76a8-7cc5-4acf-bdba-50ffaa6ee453)

![image](https://github.com/user-attachments/assets/92da56ec-c452-4674-922f-1de11356ea63)

2) Media enquiries with horizontal scroll bar

![image](https://github.com/user-attachments/assets/8240a1fe-5762-48be-86cd-a40dee16c5ea)

**Comment by zimw7** [comment]
Issue fixed.
